news/news.rst

```diff
@@ -0,0 +1,25 @@
+**Added:**
+
+* Users can use the environment variable "TEST_XPDACQ_BASE_DIR" to set the base directory
+
+**Changed:**
+
+* Shutter configure for PDF beamline changes to "Close" and "Open"
```

Comment:
this will compile better if you don't leave blank lines betwen items but you indent a couple of spaces, i.e., 
```
**Changed:**
  * Shutter configure for PDF beamline changes to "Close" and "Open" Attach files by dragging & dropping, selecting or pasting them.
  * Experiment changes to version 2 databroker

Reply:
If my memory servers me well, the rever update their template of the template.rst and I am following the new template.rst. I fogort to add this new template.rst in the news folder. I will add it in a new pr.